File: Plugins/Aspose_Email_Java_for_PHP/src/aspose/email/ProgrammingOutlook/WorkingWithOutlookPersonalStorage/StringSearchInPST.php
<?php
namespace Aspose\Email\ProgrammingOutlook\WorkingWithOutlookPersonalStorage;

use com\aspose\email\PersonalStorage as PersonalStorage;
use com\aspose\email\FileFormatVersion as FileFormatVersion;
use com\aspose\email\StandardIpmFolder as StandardIpmFolder;
use com\aspose\email\MapiMessage as MapiMessage;
use com\aspose\email\MailMessage as MailMessage;
use com\aspose\email\MailQueryBuilder as MailQueryBuilder;

class StringSearchInPST{

    public static function run($dataDir=null){

        $personalStorage=new PersonalStorage();
        $fileFormatVersion=new FileFormatVersion();
        $pst = $personalStorage->create($dataDir . "search.pst", $fileFormatVersion->Unicode);

        $standardIpmFolder=new StandardIpmFolder();
        $fi = $pst->createPredefinedFolder("Inbox", $standardIpmFolder->Inbox);

        $mapiMessage=new MapiMessage();
        $mailMessage=new MailMessage();
        $fi->addMessage($mapiMessage->fromMailMessage($mailMessage->load($dataDir . "search.pst")));

        $builder = new MailQueryBuilder();
        $builder->getFrom()->contains("automated", true);

        $query = $builder->getQuery();
        $coll = $fi->getContents($query);

        print "Total Results:" . (string)$coll->size();

    }
}
?>

File: Plugins/Aspose-Email-Java-for-Jython/asposeemail/ProgrammingOutlook/StringSearchInPST.py
from asposeemail import Settings
from com.aspose.email import MapiMessage
from com.aspose.email import NoteColor
from com.aspose.email import PersonalStorage
from com.aspose.email import FileFormatVersion
from com.aspose.email import StandardIpmFolder

from java.util import Date
from java.util import Calendar

class StringSearchInPST:

    def __init__(self):
                        
        dataDir = Settings.dataDir + 'ProgrammingOutlook/WorkingWithOutlookPersonalStorage/StringSearchInPST/'
        
        personalStorage=PersonalStorage()
        fileFormatVersion=FileFormatVersion
        pst = personalStorage.create(dataDir + "search.pst", fileFormatVersion.Unicode)

        standardIpmFolder=StandardIpmFolder
        fi = pst.createPredefinedFolder("Inbox", standardIpmFolder.Inbox)

        mapiMessage=MapiMessage()
        mailMessage=MailMessage()
        fi.addMessage(mapiMessage.fromMailMessage(mailMessage.load(dataDir + "search.pst")))

        builder = MailQueryBuilder()
        builder.getFrom().contains("automated", true)

        query = builder.getQuery()
        coll = fi.getContents(query)

        print "Total Results:" . coll.size()
        
 
if __name__ == '__main__':        
    StringSearchInPST()